sdk-docs/how-to/flask.md

```diff
@@ -0,0 +1,106 @@
+# Add an environment variable
+
+A charm configuration can be added if you require environment variables, for
+example, to pass a token for a service. For example, add the following
+configuration in `charmcraft.yaml`:
+
+```yaml
+config:
+  options:
+    TOKEN:
+      description: The token for the service.
+      type: string
+      required: true
+```
+
+PaaS App Charmer will map this as the `FLASK_TOKEN` environment variable. In
+general, the environment variable name is `CHARM<config option name>`. The
```

Comment:
I find `CHARM<config option name>` confusing here. First of all I think it's `CHARM_<config option name>` isn't it? But also, it's not quite clear I think that by "in general" you mean for any supported PaaS App Charmer framework. Maybe "For any supported PaaS App Charmer framework, the environment variable would be available as `${FRAMEWORK}_${config-option}`"?

Reply:
I also think that we shouldn't comment here on the PaaS App Charmer general case (i.e., across frameworks) since thats not really relevant for most users, they probably are just interested in "how does it work for flask" or whatever framework they are using